content/docs/user-guide/assets/pipeline/scan-directories.md

```diff
@@ -6,12 +6,14 @@ description: Scan directories are monitored by Asset Processor for new and updat
 weight: 300
 toc: true
 ---
+
+# What are scan directories?
```

Comment:
The page title serves as the top level heading, so suggest this heading be removed.

```suggestion
```